workflows/ncn/hooks/before-all/install-csi.yaml

```diff
@@ -38,7 +38,7 @@ spec:
     total=120
     sleep=5
     while true; do
-      http_code=$(curl -k -s -S -o /dev/null -w '%{http_code}' https://packages.local/service/rest/v1/status || true)
```

Comment:
Nexus starts ti return HTTP code 200 from `/status` endpoint a few seconds earlier, before it can actually serve artifacts. This causes sporadic errors during upgrade.  Let's check readiness of `noos` repo instead of status.